test/data/zoomify/ImageProperties.xml

```diff
@@ -0,0 +1 @@
+<IMAGE_PROPERTIES WIDTH="1000" HEIGHT="1000" NUMTILES="21" NUMIMAGES="1" VERSION="1.8" TILESIZE="256" />
```

Comment:
Yes, this information seems to be unnecessary. The number of tiles can be calculated and so far I don't know how to create zoomify data with NUMIMAGES >= 1.